frontend/libs/studio-components-legacy/src/components/StudioExpression/converters/complexToSimpleExpression.test.ts

```diff
@@ -16,6 +16,7 @@ import { KeyLookupFuncName } from '../enums/KeyLookupFuncName';
 import { InstanceContext } from '../enums/InstanceContext';
 import { LogicalTupleOperator } from '../enums/LogicalTupleOperator';
 import { SimpleSubexpressionValueType } from '../enums/SimpleSubexpressionValueType';
+import { PredefinedGatewayAction } from '../enums/PredefinedGatewayAction';
 
 describe('complexToSimpleExpression', () => {
   it.each([true, false])('Returns true when the expression is %s', (value) => {
```

Comment:
Dette er utenfor dine endringer, men det ser ut som at denne testbeskrivelsen er feil.
```suggestion
  it.each([true, false])('Returns the boolean value when the expression is %s', (value) => {
```